tools/Az.Tools.Predictor/Az.Tools.Predictor/Az.Tools.Predictor.psd1

```diff
@@ -41,8 +41,10 @@ For more information on Az Predictor, please visit the following: https://aka.ms
 PowerShellVersion = '7.2'
 
 # Modules that must be imported into the global environment prior to importing this module
-# "RequiredModules" requires the module in the build pipeline to build/sign this package. Instead, we will validate the dependencies at runtime by Az.Tools.Predictor.psm1
+RequiredModules = @(@{ModuleName = 'PSReadLine'; ModuleVersion = '2.2.2'; },
+                    @{ModuleName = 'Az.Accounts'; ModuleVersion = '2.7.5';})
```

Comment:
Is dependency on Az.Accounts needed? Az.ToolsPredictor doesn't use anything directly from Az.Accounts.

Reply:
It needs share assembly imported by Az.Accounts.